Documentation/EyeTracking/EyeTracking_ExamplesOverview.md

```diff
@@ -49,11 +50,11 @@ You should see several gems appear and should see the scene menu at the top.
 You should notice a small semitransparent circle at the center of your Game view. 
 This acts as an indicator of your _simulated eye gaze_.
 You can move it by pressing down the _right mouse button_ and move the mouse.
-When the cursor is hovering over the gems, you will notice that the cursor will snap to the center of the currently looked at gem. 
+When the cursor is hovering over a gem, you will notice that the cursor will snap to the center of it. 
 This is a great way to test if events are triggered as expected when _"looking"_ at a target. 
-Please beware though that the _simulated eye gaze_ via mouse control is a rather poor supplement to our rapid and unintentional eye movements. 
-It's great for testing the basic functionality though before iterating on the design by deploying it to the HoloLens 2 device.
-Coming back to our sample scene: The gem rotates as long as being looked at and can be destroyed by "looking" at it and ...
+Please be aware that the _simulated eye gaze_ via mouse control is a rather poor supplement to our rapid and unintentional eye movements. 
```

Comment:
Beware was chosen intentionally as it indicates a risk that the user should be aware of.

Reply:
But if it is a risk the user should be aware of then why not use "be aware"? Besides, "beware" implicates danger.